source/scpp/src/xdr/Stellar-SCP.h

```diff
@@ -10,8 +10,6 @@
 #include "xdr/Stellar-types.h"
 
 namespace stellar {
-
-using NodeID = PublicKey;
```

Comment:
Does it need to be `using NodeID = Hash;`?

Reply:
I didn't switch it to `Hash`, it's still a `PublicKey` (`uint256`) under the hood, so your PR is still needed, but will be much simpler after this.

Reply:
This was moved to `Agora-types.h` BTW, `using NodeID = uint256;`

Reply:
Got it. Ty.